Set library install name on OS X

Libraries on OS X contain their own installation path as so-called "install
name". Set this to the installation path to make sure wkhtmltopdf can find its
libraries regardless of where it is installed.

Upstream-Status: Submitted [https://github.com/wkhtmltopdf/wkhtmltopdf/pull/3022]
--- src/lib/lib.pro.orig	2016-06-28 18:59:19.000000000 +0200
+++ src/lib/lib.pro	2016-06-28 19:03:32.000000000 +0200
@@ -33,6 +33,10 @@
    INSTALLS += headers
 }
 
+macx {
+    QMAKE_LFLAGS_SONAME = -Wl,-install_name,$$INSTALLBASE/lib/
+}
+
 windows {
    TARGET_EXT=.dll
 }
